### ISSUE TITLE
Docker api error: reset to home but the service failed to get a11y tree

### ISSUE THREAD

**Issue by xuzhang37** [google-research/android_world#314]
hi, I just got the below error in the docker cointainer when LLM finished a task and reset to home. It showed that the service failed to get a11y tree. Could you help me to figure it out? Big thanks~

```
INFO:     172.17.0.1:59470 - "POST /reset?go_home=True HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/android_world/env/android_world_controller.py", line 218, in get_a11y_forest
    return self._get_a11y_forest()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/android_world/env/android_world_controller.py", line 211, in _get_a11y_forest
    return get_a11y_tree(self._env)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/android_world/env/android_world_controller.py", line 102, in get_a11y_tree
    raise RuntimeError('Could not get a11y tree.')
RuntimeError: Could not get a11y tree.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/uvicorn/protocols/http/h11_impl.py", line 403, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/usr/local/lib/python3.11/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/usr/local/lib/python3.11/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/usr/local/lib/python3.11/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/usr/local/lib/python3.11/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/usr/local/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/usr/local/lib/python3.11/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/usr/local/lib/python3.11/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/usr/local/lib/python3.11/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/usr/local/lib/python3.11/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/fastapi/routing.py", line 301, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/server/android_server.py", line 92, in reset
    app_android_env.reset(go_home=go_home)
  File "/android_world/env/interface.py", line 238, in reset
    return _process_timestep(self.controller.reset())
                             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/android_env/wrappers/base_wrapper.py", line 38, in reset
    timestep = self._process_timestep(self._env.reset())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/android_world/env/android_world_controller.py", line 244, in _process_timestep
    forest = self.get_a11y_forest()
             ^^^^^^^^^^^^^^^^^^^^^^
  File "/android_world/env/android_world_controller.py", line 225, in get_a11y_forest
    return self._get_a11y_forest()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/android_world/env/android_world_controller.py", line 211, in _get_a11y_forest
    return get_a11y_tree(self._env)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/android_world/env/android_world_controller.py", line 102, in get_a11y_tree
    raise RuntimeError('Could not get a11y tree.')
RuntimeError: Could not get a11y tree.
```

**Comment by LeeOrange-is-me** [comment]
``` APL
 python run.py   --suite_family=android_world  --agent_name=autodroid  > test_noattack
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
E0000 00:00:1753758685.396434 1940433 trace.cc:89] Unknown tracer: none
W0729 11:11:43.869219 140629640917632 task_eval.py:121] Skipping app snapshot loading : Snapshot not found in /data/data/android_world/snapshots/com.dimowner.audiorecorder.
W0729 11:12:45.851319 140629640917632 task_eval.py:121] Skipping app snapshot loading : Snapshot not found in /data/data/android_world/snapshots/com.dimowner.audiorecorder.
W0729 11:12:46.406904 140629640917632 task_eval.py:121] Skipping app snapshot loading : Snapshot not found in /data/data/android_world/snapshots/com.dimowner.audiorecorder.
W0729 11:12:48.600553 140629640917632 android_world_controller.py:100] Could not get a11y tree, retrying.

```

may be the same error...
